ftw/news/browser/templates/news_listing_block.pt

```diff
@@ -14,7 +14,7 @@
            i18n:translate="">No content available</p>
 
         <ul class="news-row">
-            <li class="news-item" tal:repeat="item news">
```

Comment:
Please fix the selector in the test: https://github.com/4teamwork/ftw.news/blob/97153cc50d78d37727cf1b3a1a515c417a343224/ftw/news/tests/test_news_listing_block.py#L296
So that the tests pass again...